packages/material-ui-codemod/README.md

```diff
@@ -1067,6 +1068,39 @@ npx @material-ui/codemod@next v5.0.0/with-width <path>
 
 You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#material-ui-core-styles).
 
+#### `mui-replace`
+
+Replaces all `@material-ui` related prefix with `@mui`
```

Comment:
Would it be clearer?

```suggestion
Update the imports to the new package names. For instance, the `@material-ui` npm scope is now `@mui`.
```

Reply:
It doesn't just update imports but every occurence of `@material-ui`

Reply:
I have changed to
```
Replace `@material-ui` prefix (imports and dependencies) with the the new package scope `@mui`.
```

Reply:
Are you saying it could break the @material-ui/pickers and MUI X imports? Then maybe we should whitelist the transformations.

For X we moved forward without a codemod, it seemed simple enough to rename the imports.

Reply:
I believe that codemod should be a part of v5. will whitelist
- `@material-ui/pickers`
- `@material-ui/data-grid`
- `@material-ui/x-grid`
- `@material-ui/x-grid-data-generator`

Reply:
> I have changed to

> Replace every occurrence of `@material-ui` prefix with the the new package scope `@mui`.

It's not strictly true, as it's not just renaming the scope, is it? The packages' names changed beyond that i.e. `@material-ui/core` -> `@mui/material`.

Reply:
"Replace every occurrence of material-ui related package with the the new package name (listed below)"

Is it clearer?

Reply:
Yep! Much better!